### PR TITLE
New version: IObit.Uninstaller version 13.1.0.3

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Also remove version 12.4.0.7
- Related #124954

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125247)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Oops. Your pipeline failed with binary validation error.

**Comment by BrandonWanHuanSheng** [comment]
Can you check to see if there is related to the string with binary validation failure https://github.com/BrandonWanHuanSheng/InstallerMetadata/blob/main/SetupInstruction/I/Iobit/Uninstaller/string.txt. Because, later on it might failed with this expected code.

**Comment by Dragon1573** [comment]
Manually validated in Windows Sandbox. Unable to reproduce the exception.

<details><summary>Logs</summary>

```text
PS D:\Repository\winget-pkgs> .\Tools\SandboxTest.ps1 
--> Checking dependencies
--> Starting Windows Sandbox, and:
    - Mounting the following directories:
      - C:\Users\Dragon1573\AppData\Local\Temp\SandboxTest as read-only
      - D:\Repository\winget-pkgs as read-and-write
    - Installing WinGet
    - Configuring Winget
```

```text
--> Installing WinGet
--> Disabling safety warning when running installer
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> Update-EnvironmentVariables
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget settings --enable LocalManifestFiles
Enabled admin setting 'LocalManifestFiles'.
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget install -m .\manifests\i\IObit\Uninstaller\13.1.0.3\
Found IObit Uninstaller [IObit.Uninstaller] Version 13.1.0.3
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://cdn.iobit.com/dl/iobituninstaller.exe
  ██████████████████████████████  27.1 MB / 27.1 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

image

image

</details>

**Comment by Dragon1573** [comment]
@BrandonWanHuanSheng - There's a [`BlockingDetectionFound`](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=511943&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744&l=35) error in the Azure Pipeline. What is that?

File: manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART /Install
+  SilentWithProgress: /SILENT /NORESTART /Install
+Installers:
+- Architecture: x86
```
Comment (BrandonWanHuanSheng) [inline manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml:17]:
```suggestion
- Architecture: x64
```

**Review by BrandonWanHuanSheng — COMMENTED** [review]
> There's a [`BlockingDetectionFound`](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=511943&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744&l=35) error in the Azure Pipeline. What is that?

It seem like setup can only be run from x64. But could be x86 having that error.

**Comment by BrandonWanHuanSheng** [comment]
Seem like two windows is using x64 bits but the blocking detection found on x86 machine.

**Comment by BrandonWanHuanSheng** [comment]
It seem coming from C:\Program Files (x86)\Iobit\IObit Uninstaller\vcl120.bpl

**Comment by BrandonWanHuanSheng** [comment]
It might being something related to windows. Validation  still being progress.

**Comment by Dragon1573** [comment]
What should I do to fix this problem? This package can be successfully installed on my device.

- https://github.com/microsoft/winget-pkgs/pull/125247#issuecomment-1803769495

**Comment by BrandonWanHuanSheng** [comment]
I will use Virtual Machine to see if that help.

**Comment by BrandonWanHuanSheng** [comment]
It seem like we need to set ElevationRequirement: elevatesSelf.

File: manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /VERYSILENT /NORESTART /Install
+  SilentWithProgress: /SILENT /NORESTART /Install
```
Comment (BrandonWanHuanSheng) [inline manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml:15]:
```suggestion
  SilentWithProgress: /SILENT /NORESTART /Install
ElevationRequirement: elevatesSelf
```

File: manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+Installers:
+- Architecture: x64
```
Comment (BrandonWanHuanSheng) [inline manifests/i/IObit/Uninstaller/13.1.0.3/IObit.Uninstaller.installer.yaml:17]:
```suggestion
- Architecture: x86
```

**Review by BrandonWanHuanSheng — COMMENTED** [review]
> What should I do to fix this problem? This package can be successfully installed on my device.
> 
> - https://github.com/microsoft/winget-pkgs/pull/125247#issuecomment-1803769495

Seem like it is must to have admin privileged able to install. Due to two of them blocking detection found is non admin privileged.
@Dragon1573 So, it could not be arch issue.

**Comment by Dragon1573** [comment]
Seems @wingetbot is doing the same work in #126088 ...

This one can be closed.

**Review by BrandonWanHuanSheng — COMMENTED** [review]
Seem like This one caused the Installer Scan Failure
image

**Pull request closed without merging**